app/src/main/java/org/glucosio/android/tools/ReadingToCSV.java

```diff
@@ -30,22 +30,15 @@
 import java.io.File;
 import java.io.FileOutputStream;
 import java.io.OutputStreamWriter;
-import java.util.ArrayList;
+import java.util.List;
 
 import io.realm.Realm;
 
 public class ReadingToCSV {
 
-    private Context context;
-
-    public ReadingToCSV(Context mContext) {
-        this.context = mContext;
-    }
-
-
-    public String createCSVFile(Realm realm, final ArrayList<GlucoseReading> readings, String um) {
+    public static String createCSVFile(Context context, Realm realm, final List<GlucoseReading> readings, String um) {
```

Comment:
Its not required to instantiate the class/object for the current usage, you only wish to "consume" code residing in it, which really makes it more of a **helper** class.
That being said I should've declared the class final, will do. 
This way you ensure fewer resources since the system will create the reference in compile time instead of runtime.

http://stackoverflow.com/a/218761/3374428
http://stackoverflow.com/a/5181618/3374428